app/src/main/java/edu/uc/groupproject/covid19tracker/ui/main/ByCountryFragment.kt

```diff
@@ -38,9 +37,9 @@ class ByCountryFragment : Fragment() {
          * Declare field and other necessary variables
          */
         val view = inflater.inflate(R.layout.fragment_by_country, container, false)
-        val countryListView: ListView = view.findViewById(R.id.country_list_view) as ListView
-        val countrySpinner: Spinner = view.findViewById(R.id.country_spinner) as Spinner
-        val refreshView: SwipeRefreshLayout = view.findViewById(R.id.by_country_refresh_view) as SwipeRefreshLayout
+        val countryListView: ListView = view.findViewById(R.id.country_list_view)
+        val countrySpinner: Spinner = view.findViewById(R.id.country_spinner)
+        val refreshView: SwipeRefreshLayout = view.findViewById(R.id.by_country_refresh_view)
```

Comment:
Removed some redundancy https://kotlinlang.org/docs/reference/coding-conventions.html

Reply:
Yes... with Kotlin synthetics, you don't need to specify type.